tools/tuf/test/create_repository.sh

```diff
@@ -192,6 +225,21 @@ for system in $SYSTEMS; do
         done
     fi
 
+    # Add extensions on linux (if set).
+    if [[ $system == "linux-arm64" && -n "$LINUX_TEST_EXTENSIONS" ]]; then
+        for extension in ${LINUX_TEST_EXTENSIONS//,/ }
+        do
+            extensionName=$(basename $extension)
+            extensionName=$(echo "$extensionName" | cut -d'.' -f1)
+            GOARCH="arm64" ./build/fleetctl updates add \
```

Comment:
You will need to add the corresponding build of the hello world extensions in `tools/test_extensions/hello_world/build.sh`

Reply:
Added 👍